SUMMARY.md

```diff
@@ -85,32 +106,59 @@
+* [How Do I Create a Custom Schema Field Type?](tutorials/howtos/custom-schema-field-types.md)
 * [Sending email from your ApostropheCMS site](tutorials/devops/email.md)
 * [Storing Sessions in Redis](tutorials/devops/storing-sessions-in-redis.md)
 * [Running Apostrophe on multiple cores and/or servers](tutorials/devops/multicore.md)
 * [Running Apostrophe on Windows](tutorials/devops/windows.md)
 * [Building Docker images for Apostrophe projects](tutorials/devops/docker.md)
 * [Apostrophe Migration Guide](tutorials/devops/migration.md)
-
-## HOWTOS
-
-* [Apostrophe HOWTOs](tutorials/howtos/README.md)
 * [Building a contact form in Apostrophe](tutorials/howtos/forms.md)
 * [Building combined menus and custom buttons with the admin bar](tutorials/howtos/admin-bar.md)
 * [How do I create a '404 not found' page?](tutorials/howtos/how-do-i-create-a-404-not-found-page.md)
 * [How do I add an Intranet to My Site?](tutorials/howtos/intranet.md)
 * [Configuring CKEditor in Apostrophe](tutorials/howtos/ckeditor.md)
 * [Facebook open graph tags in Apostrophe](tutorials/howtos/facebook.md)
 * [Sending user notifications](tutorials/howtos/sending-user-notifications.md)
-* [Logging in via Google, Twitter, Facebook and other sites](tutorials/howtos/google-twitter-and-facebook-login.md)
+* [How do I Redirect a User on Login?](tutorials/howtos/user-redirect.md)
 
-## Other
+## Reference
 
 * [Glossary](other/glossary.md)
+* [Schema Field Types](other/field-types.md)
+  * [area](other/field-types/area.md)
+  * [array](other/field-types/array.md)
+  * [attachment](other/field-types/attachment.md)
+  * [boolean](other/field-types/boolean.md)
+  * [checkboxes](other/field-types/checkbox.md)
+  * [color](other/field-types/color.md)
+  * [date](other/field-types/date.md)
+  * [email](other/field-types/email.md)
+  * [float](other/field-types/float.md)
+  * [integer](other/field-types/integer.md)
+  * [joinByArray](other/field-types/joinByArray.md)
+  * [joinByArrayReverse](other/field-types/joinByArrayReverse.md)  
+  * [joinByOne](other/field-types/joinByOne.md)
+  * [joinByOneReverse](other/field-types/joinByOneReverse.md)
+  * [object](other/field-types/object.md)
+  * [password](other/field-types/password.md)  
+  * [range](other/field-types/range.md)
+  * [select](other/field-types/select.md)  
+  * [singleton](other/field-types/singleton.md)
+  * [string](other/field-types/string.md)
+  * [slug](other/field-types/slug.md)
+  * [tags](other/field-types/tags.md)
+  * [time](other/field-types/time.md)
+  * [url](other/field-types/url.md)  
+  * [video](other/field-types/video.md)  
+* [Schema Field Properties](other/field-properties.md)
+  * [choices](other/properties/choices.md)
+  * [options](other/properties/options.md)
+  * [sortify](other/properties/sortify.md)
 * [Core Browser](other/core-browser.md)
 * [Core Server](other/core-server.md)
 
-## Modules
+## Apostrophe Module Reference
```

Comment:
We have "Reference" and "Apostrophe Module Reference". Should this be tiered under "Reference" as "Apostrophe Modules"?

Reply:
Moved this around to make more sense organizationally.